docs/guide/in-depth/code-splitting.md

```diff
@@ -169,12 +174,12 @@ console.log(foo);
 
 ### Respecting Execution Order doesn't take precedence
 
-- However, rolldown sometimes places modules without respecting their original order, because ensuring modules are singleton takes precedence over placing modules in the order they are declared.
+However, Rolldown sometimes places modules without respecting their original order. This is because ensuring modules to be singleton takes precedence over placing modules in the order they are declared.
```

Comment:
```suggestion
However, Rolldown sometimes places modules without respecting their original order. This is because ensuring that modules are singletons takes precedence over placing modules in the order they are declared.
```